software/index.html

```diff
@@ -36,96 +37,139 @@
-                        <img src="./images/logo_white.jpg" alt="Software Saturdays Logo" width="300">
-                        <h1 class="display-3 fw-bold">Software Saturdays</h1>
-                        <p class="fs-4">Sponsored by the Purdue College of Engineering</p>
+                    
+                    <h1 class="display-1 fw-bold text-light">Software Saturdays</h1>
+                     <!-- Registration Buttons -->
+                    <div class="clearfix">
+                            <a href="https://forms.gle/C5XQDh6ATyFnwfdX7" target="_blank" type="button" class="btn bg-purdue btn-lg fs-2 float-center mt-3 mx-1 px-4">S23 Student Registration</a>
+                            <a href="https://forms.gle/PNS4h3NUN37c7X7D7" target="_blank" type="button" class="btn btn-secondary btn-lg fs-2 float-center mt-3 mx-1 px-4">S23 Mentor Application</a>
                     </div>
                 </div>
-            </div>
-
-            <div class="my-2 container-fluid bg-pieee">
-                <div class="container-lg p-5">
 
-                    <h2 class="fs-1">Teaching modern software tools to the modern Purdue engineer.</h2>
-                    <p class="fs-5 text-start">
-                        Purdue IEEE is partnering with the Purdue College of Engineering to offer Software Saturdays, a FREE course available to all Purdue students!
-                    </p>
-                    <p class="fs-5 text-start">
-                        Software Saturdays introduces software development and programming frameworks to our attendees through a project-based learning experience.
-                        We introduce concepts and projects that are useful regardless of major: in the 21st century, having basic programming knowledge is vital no matter your hobbies or jobs you pursue.
-                    </p>
-                    <p class="fs-5 text-start">
-                        Sponsored by the Purdue College of Engineering, we are able to offer an official Purdue College of Engineering certification for everyone who attends!
+             <!-- Basic Info -->
+            <div class="my-3 p-5 container-lg fade-in-left">
+                <div class="d-flex flex-column">
+                    <h2 class="fs-1 text-light">Software Development <strong class="purdue-text">Simplified</strong>.</h2>
+                    <br/>
+                    <p class="fs-4 text-start text-light para-width">
+                         Software Saturdays is a free 8-week course, sponsored by Purdue IEEE and the College of Engineering, covering the basics of a software development framework, rotating every semester. This program is for any Purdue student who is seeking to learn a new software development framework, spruce up their resume, or is just interested in software. Additionally, everyone can earn a <strong class="purdue-text">College of Engineering</strong> certification upon completion. These look great on LinkedIn pages and resumes alike.
```

Comment:
This is begging for an image on the right, it looks off without it.

Reply:
Added image and had it get removed for mobile since it just doesn't fit well on a mobile page